GUI/GUI.csproj

```diff
@@ -50,47 +50,47 @@
     <ApplicationManifest>app.manifest</ApplicationManifest>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Chemistry, Version=1.0.0.0, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\packages\mzLib.1.0.319\lib\net471\Chemistry.dll</HintPath>
+    <Reference Include="Chemistry, Version=1.0.0.0, Culture=neutral, processorArchitecture=AMD64">
```

Comment:
does this processor architecture matter? I doubt it but just asking in case...

Reply:
I think this comes from building it as x64 at some point. MSIL is apparently for x32 and x64... I'm pretty sure this can be changed back.